### ISSUE TITLE
Centered layout not set when reopening editor

### ISSUE THREAD

**Issue by Krizz** [zed-industries/zed#22360]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When turning  "toggle centered layout" on and then reopening Zed the layout is not centered. 

### Zed Version and System Specs

Zed: v0.166.1 (Zed)
OS: macOS 15.0.1
Memory: 8 GiB
Architecture: aarch64

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

```

```
</details>

**Comment by osiewicz** [comment]
This works for me on the same version. Are you opening the same workspace that the centered layout was toggled for?

**Comment by Krizz** [comment]
Is this not something that can be set globally? Here's the video to show what is happening. 

https://github.com/user-attachments/assets/a1d47713-ead5-4a1c-a8fe-b92d6e5f8796

**Comment by osiewicz** [comment]
No, we don't support centering layout by default for new projects at the moment.

**Comment by yajo** [comment]
I think it's not a matter of project, but of ergonomics. I don't mind what project am I editing, my neck is always the same.

At least, it would be nice to have a `centered_layout.enabled_by_default` setting available, that lets us choose that default layout forever. However, the best would be that zed just remembers last state.

Thanks!
